packages/trees/frontend/sirius-components-trees/src/treeitems/TreeItem.tsx

```diff
@@ -331,9 +340,33 @@ export const TreeItem = ({
       />
     );
   } else {
+    let itemLabel: JSX.Element;
+    const splitLabelWithTextToHighlight: string[] = item.label.split(highlightRegExp);
+    if (textToHighlight === '' || splitLabelWithTextToHighlight.length === 1) {
+      itemLabel = <>{item.label}</>;
+    } else {
+      const languages: string[] = Array.from(navigator.languages);
```

Comment:
While it works, the local in Sirius Components is technically enforced by the backend so we may compare with the locale of the frontend and it may not create any issue 99% of the time, it's not really accurate. It's not really an issue right now but it may trigger some funny bug later (what would a Japanese locale do to compare English strings? or the other way around)